src/fidesops/schemas/dataset.py

```diff
@@ -62,14 +62,18 @@ class FidesCollectionKey(ConstrainedStr):
     def validate(cls, value: str) -> str:
```

Comment:
`# This overrides the default method to throw the custom FidesValidationError`
no longer accurate